libexec/pyenv-init

```diff
@@ -109,7 +109,7 @@ function detect_profile() {
     profile='~/.zprofile'
     rc='~/.zshrc'
     ;;
-  ksh )
+  ksh | ksh93 | ksh88 | pdksh | mksh )
```

Comment:
Maybe this warrants a brief comment explaining where the rarer flavors can be found -- so that at a later point, we can assess if we can drop some.

Reply:
In that case, it may make sense to leave out ksh88 and pdksh right away, as neither is active. Happy to go either way.

Reply:
Not sure what you mean here by "active". We shouldn't have code that won't be used.

Reply:
OK. I looked at a couple more distro package manifests to confirm, added a comment that I think reflects the reality of the situation, and removed the obsolete cases.